linux_os/guide/services/ssh/ssh_server/sshd_enable_strictmodes/oval/shared.xml

```diff
@@ -16,27 +17,43 @@ and configurations.</description>
       <criteria comment="sshd is not installed" operator="AND">
         <extend_definition comment="sshd is not required or requirement is unset"
         definition_ref="sshd_not_required_or_unset" />
+        {{% if product in ['opensuse', 'sle11', 'sle12'] %}}
+        <extend_definition comment="rpm package openssh removed"
+        definition_ref="package_openssh_removed" />
+        {{% else %}}
         <extend_definition comment="rpm package openssh-server removed"
         definition_ref="package_openssh-server_removed" />
+        {{% endif %}}
       </criteria>
       <criteria comment="sshd is installed and configured" operator="AND">
         <extend_definition comment="sshd is required or requirement is unset"
         definition_ref="sshd_required_or_unset" />
+        {{% if product in ['opensuse', 'sle11', 'sle12'] %}}
+        <extend_definition comment="rpm package openssh installed"
+        definition_ref="package_openssh_installed" />
+        {{% else %}}
         <extend_definition comment="rpm package openssh-server installed"
         definition_ref="package_openssh-server_installed" />
+        {{% endif %}}
         <criterion comment="Check StrictModes in /etc/ssh/sshd_config"
         test_ref="test_sshd_enable_strictmodes" />
       </criteria>
     </criteria>
   </definition>
+{{% if product in ['opensuse', 'sle11', 'sle12'] %}}
+  <ind:textfilecontent54_test check="all" check_existence="all_exist"
+  comment="tests the value of StrictModes setting in the /etc/ssh/sshd_config file"
+  id="test_sshd_enable_strictmodes" version="1">
+{{% else %}}
   <ind:textfilecontent54_test check="all" check_existence="none_exist"
```

Comment:
you can delete this `else` statements and use only OVAL for `all_exist` in all products which is the correct version to match the `StrictModes yes` into the configuration file.

Reply:
Done, and also fixed some of the descriptions in the corresponding rule.yml.